utils.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
+package houdini
+
+import (
+	"os"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"syscall"
+
+	"github.com/cloudfoundry-incubator/garden"
+)
+
+func RunCommandAsUser(cmd *exec.Cmd, spec garden.ProcessSpec) error {
```

Comment:
no need for this to be exported; mind lowercasing?

also rather than `runCommandAsUser` can this just be called `setUser`? it doesn't actually run the command.